docs: add info about working groups (#1075)
asyncapi/community: WORKING_GROUPS.md

### Definition of a Working Group

A Working Group is a group of people in our community who share a common interest beyond GitHub repositories.

A Working Group must have at least one of its members appointed as its chairperson.

Anyone can create a Working Group. The only prerequisites are to have at least 3 founding members and the approval of the TSC (Technical Steering Committee).

A Working Group should have clearly defined goals that are easily accessible to everyone on the internet, as this helps maintain alignment within the group.

> The Working Group should have an associated project management tool like [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/creating-a-project) (or any other project management tool) to outline their roadmap. It's up to the Working Group to decide the structure of the project. E.g., Kanban, Shape Up, TO-DO list, etc.

Author: ```suggestion
It is recommended that the Working Group has an associated project management tool like [GitHub Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/creating-a-project) (or any other project management tool) to outline their roadmap. It's up to the Working Group to decide the structure of the project. E.g., Kanban, Shape Up, TO-DO list, etc.
```
